vendors/twtg/models/neon-ts/model.yaml

```
@@ -31,11 +31,20 @@ LoRaWANCertified: false
 # Always mandatory: `<vendorId>:<modelName>:<modelVersion>` (You should have only one per model)
 modelId: twtg:neon-ts:1
 # <vendorId>:<protocolName>:<protocolVersion> Needed for linking the model with a specific driver -> must be the same one used in driver.yaml in the corresponding model (You might have several ones)
-protocolId:
+protocolId: twtg:ts:1
 # DataSheet URL (optional)
 specificationURL: https://github.com/TWTG-R-D-B-V/neon-product-documentation/blob/main/TS/TWTG-NEON_Temperature-Sensor_Product-Sheet_B1.pdf
```

Although it is correct, I prefer to have the link to the parent folder, such that on document version updates we don't need to update the links everywhere
```suggestion
specificationURL: https://github.com/TWTG-R-D-B-V/neon-product-documentation/tree/main/TT
```

---

Corrected on Temperature Sensor